api.go

```diff
@@ -444,3 +443,22 @@ func (client *Client) PublishVolume(volumeId string, initiatorName string) (stri
 	klog.Infof("successfully mapped volume (%s) for initiator (%s) using LUN (%d)", volumeId, initiatorName, lun)
 	return strconv.Itoa(lun), nil
 }
+
+// GetVolumeWwn: Retrieve the WWN for a volume, very useful for host operating system device mapping
+func (client *Client) GetVolumeWwn(volumeName string) (string, error) {
+
+	wwn := ""
+	response, _, err := client.ShowVolumes(volumeName)
```

Comment:
just a nit, but i think you can use the 2nd result from ShowVolumes to simplify the error checking that follows?

Reply:
Good point, I will adjust to use the return status provided.